core/trino-main/src/main/java/io/trino/execution/executor/timesharing/TimeSharingTaskExecutor.java

```diff
@@ -593,10 +593,10 @@ public void run()
                     catch (Throwable t) {
                         if (!split.isDestroyed()) { // ignore random errors due to driver thread interruption
                             if (t instanceof TrinoException trinoException) {
-                                log.error(t, "Error processing %s: %s: %s", split.getInfo(), trinoException.getErrorCode().getName(), trinoException.getMessage());
+                                log.debug(t, "Error processing %s: %s: %s", split.getInfo(), trinoException.getErrorCode().getName(), trinoException.getMessage());
```

Comment:
in `Decrease logging verbocity`

verbocity -> verbosity

maybe add more color to commit title (like "Decrease logging verbosity for split processing failures")

noun missing in "Getting exception in ??? is a non-rare faililure mode."

> Also excpetion will not be lost as it is propagated to split runner and
eventually will surfase as task failure and also possibly query failure.

Isn't that best effort?